scripts/redshift/loader.py

```diff
@@ -0,0 +1,155 @@
+import argparse
+import concurrent.futures
+import multiprocessing
+import os
+
+from matrix.common.aws.redshift_handler import RedshiftHandler
+from matrix.common.query_constructor import format_str_list
+from matrix.common.etl import (get_dss_client,
+                               etl_dss_bundles,
+                               transform_bundle,
+                               finalizer_reload,
+                               finalizer_update,
+                               upload_and_load,
+                               load_tables)
```

Comment:
I know this isn't really the style that's used throughout the rest of the project, but at some point it might be easier to just `from matrix.common import etl`